docs/src/main/sphinx/admin/properties-optimizer.md

```diff
@@ -93,6 +93,15 @@ additional synchronization when collecting results. Enabling this optimization c
 `UNION ALL` speed, when write speed is not yet saturated. However, it may slow down queries
 in an already heavily loaded system.
 
+## `optimizer.push-filter-into-values-max-row-count`
+
+- **Type:** {ref}`prop-type-integer`
+- **Default value:** `100`
+- **Minimum value:** `0`
+- **Session property:** `push_filter_into_values_max_row_count`
+ 
+Maximum number of rows in values for which filter is pushed down into values.
```

Comment:
`The number of rows in ``VALUES`` below which the planner will evaluate a filter on top of VALUES to optimize the query plan`

Reply:
Awesome .. used that with some rewording and also linking to VALUES